src/job/jobHandler.ts

```diff
@@ -297,13 +297,20 @@ export abstract class JobHandler {
   // call this method when we want benchmarks and uses cwd option to call command outside of a one liner.
   private async wrapWithBenchmarks(
     buildStepFunc: () => Promise<CliCommandResponse>,
-    stage: string
+    stage: string,
+    logger: IJobRepoLogger
```

Comment:
NIT: We don't need the logger parameter as we can access it via `this.logger` in this method.

Reply:
Great point. I'm so used to the new functions that I was not thinking straight. Thanks!